### ISSUE TITLE
Add a Notification "test" column

### ISSUE THREAD

**Issue by sastels** [cds-snc/notification-planning#35]

**Comment by yaelberger-commits** [comment]
Hey team! Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Notify-Planning-614b3ad91bc2030015ed22f5/issues/cds-snc/notification-planning/35?planning-poker) with Zenhub @andrewleith @jimleroyer @jzbahrai @sastels @smcmurtry

**Comment by jimleroyer** [comment]
Good idea, let's just name the column in a clear way and with documentation so we know its intent. We could also regenerate the database documentation afterward:
https://cds-snc.github.io/notification-documentation-database/
